### ISSUE TITLE
Is there a way to use one of these LogitsProcessors from the API?

### ISSUE THREAD

**Issue by accupham** [NVIDIA/logits-processor-zoo#6]
I see how to add it into vLLM when calling it from python, but is there a way to do it when loading the engine to serve via API?

**Comment by aerdem4** [comment]
I am not familiar with vllm serve engine API. I can have a look if you can share more details.

**Comment by mepwang** [comment]
yes. 

start the engine using:
`vllm serve xxx --logits-processor-pattern myprocessor`
myprocessor is the custom logits processor

client call API using extra_body parameter, you can refer on page: [https://docs.vllm.ai/en/stable/serving/openai_compatible_server.html](url)
Chat API

**Comment by accupham** [comment]
Thank you!
